tests/queries/0_stateless/02985_dialects_with_distributed_tables.sql

```diff
@@ -0,0 +1,27 @@
+-- Tags: distributed, replica, shard, zookeeper
```

Comment:
@alexey-milovidov I originally had this as two different functional tests with the last few lines being the only difference between the two. I ultimately decided to just combine them into this single test.

If you'd prefer this as two different tests, I can definitely split this up.

Reply:
Either variant is good for me.

Reply:
Just disable it with Fast test (no-fasttest), because PRQL is not compiled there.

Reply:
The one tag I forgot 😅